app/assets/stylesheets/navbar.css.scss

```diff
@@ -0,0 +1,59 @@
+@import url('https://fonts.googleapis.com/css2?family=Quicksand:wght@400;700&display=swap');
+
+$primaryColor: #AEFEFF;
+$very_pale_cyan: #4FBDBA;
+$very_Dark_Cyan: #072227;
+$normal_Cyan: #35858B;
```

Comment:
This variables can be declared into another file to have a better structure  in this file.

Reply:
waiting for merge and extract common rules

Reply:
added import of variables

Reply:
added main.css.scss